### ISSUE TITLE
Exuviation blue spell does not erase

### ISSUE THREAD

**Issue by kiazuki** [EdenServer/community#3479]
### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

**Comment by E5una** [comment]
I’m not at a PC until morning to validate but I had a ticket like a year ago for this and the erase was fixed and it was added. Are you sure you had an erasable debuff on? Keep in mind Erase won’t remove something a -na spell takes off. Need a lot more detail in the bug report.

**Comment by kiazuki** [comment]
Hey. You would know a lot more than me. It didnt remove gravity, poison or paralyze for me. The poison was from sea. The paralyze was from sky. Gravity was from a point in sky.

**Comment by E5una** [comment]
Since Paralyna and Poisana exist, Erase does not work on Paralyze and Poison. It should work on Weight though. My guess is that you had another debuff on (maybe dia or MND down from pots if your weight came from pots?) and since Exuviation only removes one debuff, those came off instead of your weight. 

I just tested with Goblin Alchemists in Cape Terrigan. Waited to get Dia III'd, slept it, cast Exuviation and validated that it was removed.  I've definitely taken off Gravity before as well, so I believe this ticket can be closed.
